### PR TITLE
Bugfix [v2] validation and pop up

### PR DESCRIPTION
**Pull request by rahmatalmubarak**

### CODE REVIEW

**Review by 4fdal — APPROVED** [review]

File: src/Controllers/BadasoDatabaseController.php [hunk 1]
```diff
@@ -42,13 +42,30 @@ function ($attribute, $value, $fail) {
...
+                'rows.*.field_type' => [
+                    'required',
+                    'string',
+                    function ($attribute, $value, $fail) use ($request) {
+                        $request_data = $request->rows;
+
+                        $mysql_data_type = [
+                            'tinyint', 'smallint', 'mediumint', 'int', 'integer', 'bigint', 'decimal', 'float', 'double', 'bit', 'char', 'varchar', 'binary', 'varbinary', 'tinyblob', 'blob', 'mediumblob', 'longblob', 'tinytext', 'text', 'mediumtext', 'longtext', 'enum', 'set', 'date', 'time', 'datetime', 'timestamp', 'year', 'geometry', 'point', 'linestring', 'polygon', 'geometrycollection', 'multilinestring', 'multipoint', 'multipolygon', 'json', 'boolean',
+                        ];
+                        $cek = [];
```
Comment (rizkiheryandi) [inline src/Controllers/BadasoDatabaseController.php:54]: is "cek" means "check" ?

please using english for the variable

File: src/resources/js/lang/modules/id.js [hunk 2]
```diff
@@ -272,6 +273,7 @@ export default {
     },
     warning: {
       notAllowed: "Anda tidak diizinkan untuk mengakses CRUD.",
+      idNotAllowed: "Jangan ganti nama 'id' table dengan apapun",
```
Comment (rizkiheryandi) [inline src/resources/js/lang/modules/id.js:276]: table > tabel

File: src/resources/js/pages/home.vue [hunk 3]
```diff
@@ -54,12 +54,21 @@ export default {
...
-            title: this.$t("alert.danger"),
-            text: error.message,
-            color: "danger",
+         if(error.status == 401){
+            this.$closeLoader();
+            this.$vs.notify({
+              title: this.$t("alert.error"),
+              text: error.message,
+              color: "danger",
+          });
```
Comment (rizkiheryandi) [inline src/resources/js/pages/home.vue:63]: need more tab or space for indentation

**Comment by Dri372** [comment]
Hi @rahmatalmubarak 
I tested version 2.x-dev but bug is not solved, same symptom as #536

composer.lock
...
            "name": "uasoft-indonesia/badaso",
            "version": "2.x-dev",
            "source": {
                "type": "git",
                "url": "https://github.com/uasoft-indonesia/badaso.git",
                "reference": "036cf8e9c924d2ca328ef6471b1acd01c6037d9e"
            },

**Pull request merged**